Payments/Integration/01_ManageConnection.md

```diff
@@ -0,0 +1,145 @@
+
+#### Procedure    
+
+*Payments > Settings > Tab CONNECTIONS*   
+
+![Connections](../../Assets/Screenshots/Payments/Settings/Settings.png "[Connections]")
+
+1.  Click the ![Add](../../Assets/Icons/Plus01.png "[Add]") (Add) button in the bottom right corner.   
+    The *Connection* view is displayed.   
+    ![Create connection](../../Assets/Screenshots/Payments/Settings/CreateConnection.png "[Create connection]")
+
+2.  Enter a name for the connection in the *Name* field.
+
+3.  Click the *Driver* drop-down list and select the desired driver. 
+   All already installed drivers are displayed in the list.   
+   
+     > [Info] If you want to install additional drivers, select the **more...** option and buy a driver in the app store. You have to purchase drivers from the main account. You need to install them separately in each sandbox or account where you want to use it.
+
+     The *Credentials* section is displayed below the drop-down list.
+
+    ![Credentials](../../Assets/Screenshots/Payments/Settings/CreateConnectionCredentials.png "[Credentials]")
+     > [Info] Depending on the selected driver, the fields in the *Credentials* section differ. 
+
+4. Enter the required data in the fields in the *Credentials* section. Use the information you received from the payment service provider to fill the credentials.   
+
+5. Click the [SAVE] button.   
+    The connection will be created. Depending on the driver, the *Checking credentials...* notice is displayed. After a few seconds, the *Loading data...* notice is displayed.
+
+     ![Checking credentials](../../Assets/Screenshots/Payments/Settings/CheckingCredentials.png "[Checking credentials]")
+
+    The data are synchronized between Actindo and the payment service provider, if a synchronization is necessary. The view for creating connections is automatically closed when the connection has been created and synchronized. The *Edit connection* view of the newly created connection is displayed. The *Settings* tab is selected.
+      ![Credentials](../../Assets/Screenshots/Payments/Settings/EditConnectionSettings.png "[Credentials]")
+    <!---Stefan: stimmt das? Ich kann es nicht nachmachen, wenn es Verbindungen sind, die nicht direkt aktiv sind, gibt es dann buttons mit denen man aktivieren, synchronisieren kann?-->
+
+
+
+
+## Edit PSP connection
+
+Edit a connection to adjust any changed credentials, the connection name or further settings. 
+
+#### Prerequisites
+
+At least one connection has been created, see [Create a connection](#create-a-psp-connection).
+
+#### Procedure
+
+*Payments > Settings > Tab CONNECTIONS*
+
+![Connections](../../Assets/Screenshots/Payments/Settings/Settings.png "[Connections]")
+
+1. Select the checkbox of the driver to select the connection you want to edit.  
+The editing toolbar is displayed above the list of connections.
+
+2. Click the ![Edit](../../Assets/Icons/Edit01.png "[Edit") (Edit) button.   
+    The *"Driver name" connection* view is displayed.
```

Comment:
The *Edit connection* view is displayed.

Vgl. Omni-Channel: 
https://github.com/Actindo-AG/Documentation/blob/main/Channels/Integration/01_ManageConnections.md

Reply:
done